site/content/en/references/kustomize/kustomization/patches/_index.md

```diff
@@ -55,122 +55,213 @@ The `name` and `namespace` fields of the patch target selector are
 automatically anchored regular expressions. This means that the value `myapp`
 is equivalent to `^myapp$`. 
 
-Consider the following `deployment.yaml` common for both the examples:
+With patches it is possible to override the kind or name of the resource it is editing.
+By default, the patch will leave the kind and name of the resource untouched. For example:
```

Comment:
Let's change the ordering of things for clarity. Please change this to:

With patches it is possible to override the kind or name of the resource it is editing with the options `allowNameChange` and `allowKindChange`. For example:

< example >

By default, these fields are false and the patch will leave the kind and name of the resource untouched.